src/main/java/features/Cart.java

```diff
@@ -1,4 +1,88 @@
 package features;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Cart {
+
+    private ArrayList<Listing> listings;
+
+    /** A Constructor for Cart that sets contents of the cart to be itemListings.
+     *
+     * @param itemListings an ArrayList of unique Listings.
+     */
+    public Cart(ArrayList<Listing> itemListings){
+        this.listings = itemListings;
+    }
+
+    //OVERLOADED
+    /** A Constructor for Cart that takes no arguments*/
+    public Cart(){
```

Comment:
good thinking w/ two constructors

Reply:
great documentation! makes it very clear to understand what is happening and where.